types/cli-table2/index.d.ts

```diff
@@ -0,0 +1,99 @@
+// Type definitions for cli-table2 0.2
+// Project: https://github.com/jamestalmage/cli-table2
+// Definitions by: Melvin Groenhoff <https://github.com/mgroenhoff>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare namespace CliTable2 {
+    type CharName =
+        "top" |
+        "top-mid" |
+        "top-left" |
+        "top-right" |
+        "bottom" |
+        "bottom-mid" |
+        "bottom-left" |
+        "bottom-right" |
+        "left" |
+        "left-mid" |
+        "mid" |
+        "mid-mid" |
+        "right" |
+        "right-mid" |
+        "middle";
+
+    type HorizontalAlignment = "left" | "center" | "right";
+    type VerticalAlignment = "top" | "center" | "bottom";
+
+    interface TableOptions {
+        chars: Record<CharName, string>;
+        truncate: string;
+        colWidths: Array<number | null>;
+        rowHeights: Array<number | null>;
+        colAligns: HorizontalAlignment[];
+        rowAligns: VerticalAlignment[];
+        style: {
+            "padding-left": number;
+            "padding-right": number;
+            head: string[];
+            border: string[];
+            compact: boolean;
+        };
+        head: string[];
+        wordWrap: boolean;
+    }
+
+    interface TableConstructorOptions extends Partial<Pick<TableOptions, "truncate" | "colWidths" | "rowHeights" | "colAligns" | "head" | "wordWrap">> {
```

Comment:
The `Pick` is missing `"rowAligns"`.
It might be better to just have an interface that is common to both `TableOptions` and `TableConstructorOptions`, and have them both extend from it.